### PR TITLE
fix: add action rendered trace for s3 in aggregated mode

### PR DESCRIPTION
**Pull request by kjellwinblad**
Fixes:
https://emqx.atlassian.net/browse/EMQX-12429

Manually tested

Release version: v/e5.?

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [] Added tests for the changes
- [] Added property-based tests for code which performs user input validation
- [] Changed lines covered in coverage report
- [] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [] For internal contributor: there is a jira ticket to track this change
- [] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx_bridge_s3/src/emqx_bridge_s3_connector.erl [hunk 1]
```diff
@@ -372,6 +379,13 @@ run_aggregated_upload(InstId, Records, #{aggreg_id := AggregId}) ->
             {error, {unrecoverable_error, Reason}}
     end.
 
+render_records(Records) ->
+    try
+        [unicode:characters_to_binary(R) || R <- Records]
```
Comment (keynslug) [inline apps/emqx_bridge_s3/src/emqx_bridge_s3_connector.erl:384]: Q: `Records` are maps 99.9% of time, productions of a Rule SQL expression, to be more specific. I wonder why do we attempt to stringify them like that first?

Comment (kjellwinblad) [reply apps/emqx_bridge_s3/src/emqx_bridge_s3_connector.erl:384]: You are right. This was done for non-aggregate mode so I assumed we should do the same for aggregate mode as well but I see now that this is incorrect. In non-aggregate mode we render a template. I will fix. Btw, I accidentally pushed a commit for another issue to this PR. Do you think you can review that one as well so we can merge both with the same PR?

Comment (kjellwinblad) [reply apps/emqx_bridge_s3/src/emqx_bridge_s3_connector.erl]: Removed the formatting here: [89b47e8](https://github.com/emqx/emqx/pull/13100/commits/89b47e8ffccbeff5b9149f3fc2ab9866d4e611fa)

**Review by keynslug — APPROVED** [review]
Seems reasonable! Not too confident about fb7688a due to lack of context.

**Comment by emqxqa** [comment]
[TestExecution]()

**Pull request merged**